Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -239,9 +240,15 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
             }
             catch (LibGit2SharpException exception)
             {
-                if (exception.Message.Equals("Unsupported URL protocol"))
+                tracer.TraceWarning("LibGit2SharpRepository fetch failed with {0}", exception);
+
+                // LibGit2Sharp doesn't support SSH yet. Use GitExeRepository
+                // LibGit2Sharp only supports smart Http protocol
+                if (exception.Message.Equals("Unsupported URL protocol") ||
```

Comment:
Note that this is not a libgit2sharp bug, but a deliberate decision they made not to support the old http dump protocol.

So let's try not to widely ignore exceptions from libgit2sharp so we don't mask issues.

Did we ever get the repo that caused this issue? It's quite uncommon to have servers that don't support the smart protocol.